api/routes.go

```diff
@@ -3,24 +3,28 @@ package api
 import (
 	"encoding/hex"
 	"errors"
-	"github.com/gorilla/mux"
-	"github.com/kailt/imageresizer/etag"
-	"github.com/kailt/imageresizer/imager"
-	"github.com/rcrowley/go-metrics"
+	"fmt"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"strconv"
 	"strings"
 	"unicode/utf8"
+
+	"github.com/gorilla/mux"
+	"github.com/kailt/imageresizer/etag"
+	"github.com/kailt/imageresizer/imager"
+	"github.com/rcrowley/go-metrics"
 )
 
 const uploadSizeLimit = 50 * 1024 * 1024
 
 func (api *Api) routes() {
 	api.Handle("/favicon.ico", api.handle404())
 	api.Handle("/debug/metrics", http.DefaultServeMux)
+	// shortcut
+	api.HandleFunc("/{width:[0-9]*}/{resizeOp}/{options}/{path}", api.etagMiddleware(api.serveThumbs())).Methods("GET", "HEAD")
```

Comment:
It should be `{width:[0-9]+}` because we expect at least 1 digit. This was already broken in the original code. Some non-zero validation would be cool too.